src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -86,11 +87,13 @@
   private final String appCommandOutputFormat;
   private final WaitingProcessOutputLineListener runDevAppServerWaitListener;
 
-  private CloudSdk(Path sdkPath, String appCommandMetricsEnvironment,
-      String appCommandMetricsEnvironmentVersion, @Nullable File appCommandCredentialFile,
-      String appCommandOutputFormat, ProcessRunner processRunner,
-      WaitingProcessOutputLineListener runDevAppServerWaitListener) {
+  private CloudSdk(Path sdkPath, Path javaBinaryPath, String appCommandMetricsEnvironment,
```

Comment:
The constructor isn't part of the API. The API in Builder only surfaces a javaSdkHome() method.